operator/informer_controller.go

```diff
@@ -277,6 +283,8 @@ func (c *InformerController) informerUpdateFunc(resourceKind string) func(contex
 			}
 			if err != nil && c.RetryPolicy != nil {
 				c.queueRetry(retryKey, err, func() (*time.Duration, error) {
+					ctx, span := GetTracer().Start(ctx, "controller-retry")
```

Comment:
Just some thoughts on this. Between having a separate span for retry, under the same trace like this vs aggregating all retries under one big span, I think I'd prefer having separate spans, as it feels more correct. As I understand, the retry span starts when the retry actually gets triggered from queue, which seems the right way of going about it.